arviz/stats/stats.py

```diff
@@ -326,10 +342,14 @@ def hpd(ary, credible_interval=None, circular=False, multimodal=False, skipna=Fa
         modes are well separated.
     skipna : bool
         If true ignores nan values when computing the hpd interval. Defaults to false.
+    kwargs : dict, optional
+        Additional keywords passed to ax.scatter
```

Comment:
passed to `wrap_xarray_ufunc`, maybe add a link to wrap_ufunc docs too?

 The explanation below would be better as an example instead of here in argument description I think.

Reply:
Ah, I copied it from somewhere. Sorry. I will add a few examples.